### ISSUE TITLE
How to add focal loss to replace the original loss function

### ISSUE THREAD

**Issue by yuyijie1995** [andy-yun/pytorch-0.4-yolov3#28]
Can you give me some suggestions? Thanks a lot

**Comment by andy-yun** [comment]
You can replace CrossEntropyLoss with FocalLoss. See the details in main module of focal_loss.py

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by yuyijie1995** [comment]
Thanks a lot I will try ！
